qt/dg.qrc

```diff
@@ -5,5 +5,10 @@
     <file alias="plus">../images/plus_8.png</file>
     <file alias="minus">../images/minus_8.png</file>
     <file alias="search_clear_13">../qtlib/images/search_clear_13.png</file>
+    <file alias="exchange">../images/exchange_purple_upscaled.png</file>
+    <file alias="zoom_in">../images/old_zoom_in.png</file>
+    <file alias="zoom_out">../images/old_zoom_out.png</file>
+    <file alias="zoom_original">../images/old_zoom_original.png</file>
+    <file alias="zoom_best_fit">../images/old_zoom_best_fit.png</file>
 </qresource>
```

Comment:
Commenting here, but what is the reason for all the other unused images added here?  These 5 seem to be the only ones used here.

Reply:
Indeed I left the original source files in case we need them later. These should not be included in the release package. Since the source files were lost by the author (Jason Cho), I upscaled them (with waifu2x) but left the original files in case we need them later. Turns out I'll modify that icon right now.

Reply:
Ok